Computer_Science/10_Cryptography/Caesar_cipher/2_caesar_cipher.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="1_caesar_cipher.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Ceaser Cypher</h1>
+    <form id="fid1">
+        <label for="entry">Enter Text/Code:</label><br>
+        <textarea name="entry" id="entry" rows="10" placeholder="Enter Text/Code" value=""></textarea><br>
+        <label for="key">Key:</label>
```

Comment:
Change label tag from:
<label for="key">Key:</label>
To this:
<label for="key">Key (Shift):</label>